UserMetaManagement.php

```diff
@@ -214,23 +216,23 @@ public function deleteUserMetaDetails() {
     }
 
     // Function to create a user page in the user secion in the user seciton.
-    public function addUserPage() {
-	add_users_page('User Meta Management', 'User Meta Management', 'manage_options', 'user-meta-management', array($this, 'userMetaManagement'));
+    public function UMMAddUserPage() {
```

Comment:
please dont prefix function names. They are inside a class ... so it is not an issue that they will ever clash.